src/isentropic.jl

```diff
@@ -45,13 +45,13 @@ The air pressure for an isentropic process, where
  - `p∞` ambient pressure
 """
 function air_pressure(
-    param_set::APS,
+    param_set::ThermodynamicsParameters,
     T::FT,
     T∞::FT,
     p∞::FT,
     ::DryAdiabaticProcess,
 ) where {FT <: AbstractFloat}
-    _kappa_d::FT = CPP.kappa_d(param_set)
+    _kappa_d::FT = param_set.kappa_d
     return p∞ * (T / T∞)^(FT(1) / _kappa_d)
```

Comment:
```suggestion
    kappa_d = param_set.kappa_d
    return p∞ * (T / T∞)^(FT(1) / kappa_d)
```

Reply:
On Julia 1.7, you can also do:
```suggestion
    (;kappa_d,) = param_set
    return p∞ * (T / T∞)^(FT(1) / kappa_d)
```